resource_retriever/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ if(NOT WIN32)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wpedantic")
 endif()
 
-find_package(ament_cmake_ros REQUIRED)
+find_package(ament_cmake REQUIRED)
```

Comment:
The issue with this change is that the `add_library` call will change to be !SHARED.  I'm fine with this change in combination with adding the `SHARED` keyword to `add_library`.

Reply:
Oh wow that's subtle - sure enough https://github.com/ros2/ament_cmake_ros/blob/master/ament_cmake_ros/cmake/build_shared_libs.cmake

Reply:
I put the `ament_cmake_ros` dependency back in in https://github.com/ros/resource_retriever/pull/72/commits/2bc94309b36de66087262eae2df837fd3bf8db0e

Reply:
Yeah, I don't particularly love that behavior, but that's an issue for another day.